docs/features.md

```diff
@@ -98,3 +98,10 @@ The feature defaults a service account's `automountServiceAccountToken` to false
 |7| **not defined** | false | false | false |
 |8| true | not defined | true | true |
 |9| false | not defined | false | false |
+
+## Karydia Exclusion Handling
+
+Namespaces and other objects can be opted out of being "watched" by karydia. Therefor, there are two options:
```

Comment:
Typo on "therfor". Should be "therefore".

Reply:
done